lib/core/card-view/components/card-view-textitem/card-view-textitem.component.html

```diff
@@ -9,6 +9,12 @@
                 <span class="adf-textitem-clickable-value" [attr.data-automation-id]="'card-textitem-value-' + property.key">
                     <span *ngIf="showProperty(); else elseEmptyValueBlock">{{ property.displayValue }}</span>
                 </span>
+                <button mat-icon-button fxFlex="0 0 auto" *ngIf="showClickableIcon()"
```

Comment:
missing accessibility label

Reply:
@sivakumar414ram can you add it as we have in other places ?
```
 <button mat-icon-button fxFlex="0 0 auto"
                class="adf-textitem-action"
                [attr.aria-label]="'CORE.METADATA.ACTIONS.EDIT' | translate"
                [attr.title]="'CORE.METADATA.ACTIONS.EDIT' | translate"
                [attr.data-automation-id]="'card-textitem-edit-icon-' + property.key">
```